### PR TITLE
Lb reproduce benchm mnist

### PR DESCRIPTION
**Pull request by smilesun**

### CODE REVIEW

**Comment by lisab00** [comment]
[results.csv](https://github.com/marrlab/DomainLab/files/12645660/results.csv)
stochastic_variation

80 % of the benchmark finished, manually aggregated the results

**Comment by smilesun** [comment]
@lisab00 , Could you allocate a GPU using

`srun -p gpu_p --qos=gpu_short --gres=gpu:1 --nice=10000 --pty bash`   (tested worked for me)  and work out a configuration on one specific algorithm when it would have high accuracy on mnist? 

reference can be the one we have on the documentation page of DomainLab.

**Comment by lisab00** [comment]
running this command from the examples page: "python main_out.py --te_d 1 2 --tr_d 0 3 --task=mnistcolor10 --epos=500 --bs=16 --aname=diva --nname=conv_bn_pool_2 --nname_dom=conv_bn_pool_2 --lr=1e-3 --seed=0 --gamma_y=1e5 --gamma_d=1e5 --san_check --san_num=8"
yielded only an accuracy of 50 %

**Comment by lisab00** [comment]
stochastic_variation
[results.csv](https://github.com/marrlab/DomainLab/files/12663410/results.csv)
[hyperparameters.csv](https://github.com/marrlab/DomainLab/files/12663413/hyperparameters.csv)

benchmark results: the high variation in accuracy comes from the different learning rates. the bigger lr (1e-3) performs significantly better.

**Comment by smilesun** [comment]
looks good, I have a hardtime to make my cluster work again, but will fix as well so i can also submit jobs again.
